tests/test_check_mmd.py

```diff
@@ -70,6 +72,17 @@ def test_all_urls_1(self):
         self.assertTrue(check_urls(['https://www.met.no']))
         self.assertFalse(check_urls(['http://met.not']))
 
+    # Check special urls
+    @patch('requests.get')
+    def test_special_urls_1(self, mock_get):
+        # Check urls with WMS
+        self.assertTrue(check_urls(['https://thredds.met.no/thredds/wms/remotesensingsatellite/polar-swath/2020/12/01/metopb-avhrr-20201201155244-20201201160030.nc?service=WMS&version=1.3.0&request=GetCapabilities']))
```

Comment:
problem med slike tester er at vi gjør oss avhengig av at serveren er oppe. Se på min test i tests_nc_to_mmd, linje 557 - test_file_on_thredds. Den passerer hvis serveren er nede...

Reply:
Nei, siden request.get er patchet kjøres ikke selve request kallet. Men jeg har endret testen til å bare ha de strengene vi skal teste for.

Reply:
aha - la ikke merke til patchingen... sorry, det er bra!